jenkins-build.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 set -e -x
 pip install -r ~/docker-base/pre-requirements.txt
-install-requirements.py -d ~/docker-base/base-requirements.txt -r requirements.txt -r test-requirements.txt
+pip install -r requirements.txt -r test-requirements.txt
```

Comment:
I half expected `install_pinned` here because you mentioned its use in CI...

Reply:
I guess it depends what you want this script to do (it's not used by CI so I assume it's something you use in personal testing). If you just want to get the same versions of packages that would be installed by the Jenkins then `pip install` is sufficient (and avoids the need to have `install_pinned.py` on your PATH). If you want to also catch errors in requirements that would be caught by Jenkins then using `install_pinned.py` would make more sense.

Reply:
Oh... So we use `Jenkinsfile` for CI instead?

This was introduced by CAM in 2012, probably to do CI on their side. I then updated it in 2016 to match our style. Pfft.

I'll ping Anton... If he's not using it, we can just cull it.

Reply:
[All right... I'll remove it!](https://www.youtube.com/watch?v=eCM2nEBE0RY#t=2m50)

Reply:
I don't like the `.travis.yml` either :-) Are we still using those? I see the ones in `aiokatcp` and `spead2` still had recent updates.

Reply:
For aiokatcp and spead2 I made Travis the main CI, because they have applicability outside SDP and CI on Travis makes it easier for outside developers.

If it's not actually being build on Travis then you might as well delete it.